PYME/Analysis/BleachProfile/fitRecover.py

```diff
@@ -24,10 +24,9 @@
 from . import kinModels
 
 from PYME.Analysis._fithelpers import *
-
+from pylab import *
```

Comment:
We can't have from `pylab import *` at the top level of any module as it screws with tests and documentation generation, and in general we want to move away from using `from pylab import *` anywhere. To fix this, we need to change `FitTrace()` so that it has `import matplotlib.pyplot as plt`, change all the plotting functions to `plt.xxx` and change anything like `array()`, `arange()` etc ... to use the `np.array()` etc equivalents.

Reply:
pylab is now removed from `fitRecover`. There's only one function so I just stuffed the imports into it since you specificially mentioned importing in `FitTrace`. Worth pointing out that this code is undocumented and magic-number ridden.

Reply:
The undocumented and magic number ridden critique is the case for much of the stuff under BleachProfile, and some of this should really be factored out or removed. In this case most of the magic numbers match up with a specific PYMEAcquire protocol (would need to check which one). If writing from scratch, this would probably live in an IPython notebook, rather than it's own file.

For reference, it's an attempt to measure k_dark->on, based on a protocol which turns the laser on for a brief pulse (enough to push the majority of stuff into the dark state) and then lets the sample recover in the dark for varying amounts of time to assess how much has recovered. It falls into the category of stuff which:
- probably doesn't belong in PYME core
- probably worth hanging onto / revisiting at some point in the future (i.e. don't want to delete outright)
- not sufficiently important / pressing to spend much time on at present